ee/docker-ee/ubuntu.md

```diff
@@ -119,19 +119,20 @@ and update Docker Engine - Enterprise from the repository.
 
```

Comment:
Edit step 3 to read: "Temporarily add a $DOCKER_EE_URL variable into your environment (it persists only up until you log out of the session)."